clj-refactor.el

```diff
@@ -149,7 +152,10 @@ with the middleware."
 
 (defvar cljr--nrepl-ops
   '("artifact-list" "artifact-versions" "clean-ns" "configure" "find-symbol"
-    "find-unbound" "hotload-dependency" "resolve-missing"))
+    "find-unbound" "hotload-dependency" "resolve-missing" "version"))
+
+(defvar cljr-version "1.0.5-SNAPSHOT"
```

Comment:
so you need `(defvar clj-version "1.0.5-SNAPSHOT")` twice?! sry for being slow here...